Common/src/config_structure.cpp

```diff
@@ -2514,6 +2467,34 @@ void CConfig::SetConfig_Options() {
 
   /* DESCRIPTION: Multipoint mesh filenames, if using different meshes for each point */
   addPythonOption("MULTIPOINT_MESH_FILENAME");
+
+  /*--- options that are used for the output ---*/
+  /*!\par CONFIG_CATEGORY:Output Options\ingroup Config*/
+
+  /* DESCRIPTION: Type of screen output */
+  addStringListOption("SCREEN_OUTPUT", nScreenOutput, ScreenOutput);
+  /* DESCRIPTION: Type of output printed to the history file */
+  addStringListOption("HISTORY_OUTPUT", nHistoryOutput, HistoryOutput);
```

Comment:
Would it be possible to define some defaults for the history option that would preserve previous behavior? For example, to include AERO_COEFF, RMS_RES if running a fluid simulation.

I understand that we want to keep the default as simple as possible, but the python scripts (file IO ones) read the history file to get functions like LIFT, DRAG etc. A lot of them assume that AERO_COEFF/COMBO is being written out to the history file. 

Another (less invasive) way to do it would be to set those options if running from the python scripts. But this might be confusing if running the same config file from the python script results in different outputs, than if running without them.

Reply:
I am not a big fan of adding it as defaults just to preserve previous behavior. I would go for just adding it to the output fields. In the future we could think of automatically parsing the available fields, but I don't want to do that since we first have to decide on a general strategy for the whole python framework.

Reply:
Ok that's how we can do it at the moment (gonna push it soon): I wrote a small parser that goes through all output files and looks for `AddHistoryOutput`, reads the options and writes them into a python file where this information is assigned to a dictionary.  This dictionary is then used in the python framework for mapping the header and function names and to add missing fields to the `HISTORY_OUTPUT` config option. 

Note however this is a *semi-automatic* procedure: the parser has to be called at the moment every time there is a change in the output fields. But its much more flexible as what we have currently ... And the parser could be used in the  future to fully automate the procedure.

Reply:
Thank you! Yeah, the more I use this new output structure, the more I like it. So it is definitely a significant improvement. Just coming across these questions as I merge this with #780

Reply:
If something else is missing, lets solve that in a separate PR I'd say. I am going to do a last pass to solve the conflicts/reg. tests tmrw.

Reply:
I like that idea @talbring. I will also wait for this one to merge before I pull it into #790 and clean things up there (like removing the old CFL adaption routines) and fix the regressions.